### ISSUE TITLE
[en] false alarm CD_NN

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#2020]
> But 6 or above doesn't count

<img width="410" alt="Bildschirmfoto 2019-10-12 um 19 30 22" src="https://user-images.githubusercontent.com/37363/66705432-c9b2f700-ed26-11e9-99ba-2fb6676fb29f.png">

I think there are two issues here:
* the false alarm
* the plural of the noun "above" is not "aboves" but "above" (it's uncountable)

**Comment by tiff** [comment]
@MikeUnwalla I would like to remove the plural form "aboves" of "above" by adding it to the removed.txt? Would this be okay for you?

**Comment by MikeUnwalla** [comment]
@tiff , yes. As best I know, 'aboves' is not a standard English word.

**Comment by donovanrichardson** [comment]
I'm also noticing that this rule is missing a lot incorrect cases, perhaps because they involve words listed as NNP in the dictionary. For example, "I have 2 kid" is acceptable, . Is the right course of action to move "kid" as NNP to removed as well? (Also as NN:UP for that matter) 
<img width="246" alt="kid" src="https://user-images.githubusercontent.com/50427582/66959213-e8bbcc80-f037-11e9-8f02-1b4ff7c52133.png">

**Comment by MikeUnwalla** [comment]
@donovanrichardson , thanks for the report.

'kid' (lowercase k) is not NNP. Thus, the false negative is not because of Kid/NNP.

kid/NN:UN is correct. NN:UN means that the word can be both a count noun and a non-count noun. 'Kid' is both: https://www.lexico.com/en/definition/kid

**Comment by donovanrichardson** [comment]
@MikeUnwalla Aha, I've never heard of that definition for kid!
Nevertheless, I am concerned about the exception that CD_NN provides for NN:UN, because sentences including "three kid" should not be acceptable, whether the noun is uncountable or countable singular.  For reference the line with the problematic NN:UN exception is below.
` <exception postag="NN:UN?|NNP?S|JJ.*|IN|CD" postag_regexp="yes"/>`
Testing grammar.xml without the NN:UN exception, I immediately found a sentence that should be acceptable ("I'll be in my office from ten tomorrow.") but failed the test isn't. If those failures aren't too numerous, perhaps I can begin adding antipatterns for sentences that break when the NN:UN exception is removed.

**Comment by MikeUnwalla** [comment]
@donovanrichardson, 

> perhaps I can begin adding antipatterns for sentences that break when the NN:UN exception is removed.

Yes, that is fine by me. Given that you will add antipatterns, I will leave you to deal with the removal of the NN:UN exception.
